docs/content/en/integrations/parsers/file/noseyparker.md

```diff
@@ -0,0 +1,104 @@
+
+~~~
+
+If the first line is expanded, it looks like this:
+
+~~~
+{
+    "type": "finding",
+    "rule_name": "Generic Password (double quoted)",
+    "match_content": "32ui1ffdasfhu239b4df2ac6609a9919",
+    "num_matches": 2,
+    "status": null,
+    "comment": null,
+    "matches": [
+        {
+            "provenance": [
+                {
+                    "kind": "file",
+                    "path": "app/schema/config.py"
+                },
+                {
+                    "kind": "git_repo",
+                    "repo_path": "./.git",
+                    "commit_provenance": {
+                        "commit_kind": "first_seen",
+                        "commit_metadata": {
+                            "commit_id": "0ef84b84c29924b210e3576f69d1e8632948bedc",
+                            "committer_name": "Princess Leia",
+                            "committer_email": "leia@test.com",
+                            "committer_timestamp": "1685495256 +0000",
+                            "author_name": "Princess Leia",
+                            "author_email": "leia@test.com",
+                            "author_timestamp": "1685495256 +0000",
+                            "message": "first commit\n"
+                        },
+                        "blob_path": "app/schema/config.py"
+                    }
+                }
+            ],
+            "blob_metadata": {
+                "id": "0ee84b84c29924b210e3576fe9d1e8632948bedc",
+                "num_bytes": 664,
+                "mime_essence": "text/plain",
+                "charset": null
+            },
+            "blob_id": "0ee84b84c29924b210e3576fe9d1e8632948bedc",
+            "location": {
+                "offset_span": {
+                    "start": 617,
+                    "end": 660
+                },
+                "source_span": {
+                    "start": {
+                        "line": 16,
+                        "column": 17
+                    },
+                    "end": {
+                        "line": 16,
+                        "column": 59
+                    }
+                }
+            },
+            "capture_group_index": 1,
+            "match_content": "32ui1ffdasfhu239b4df2ac6609a9919",
+            "snippet": {
+                "before": "E = \"https://testwebsite.com\"\n ",
+                "matching": "API_KEY = \"32ui1ffdasfhu239b4df2ac6609a9919",
+                "after": "\"\n\n\n"
+            },
+            "rule_name": "Generic API Key"
+        }
+    ]
+}
+~~~
```

Comment:
Since this PR was opened (sorry it's taken me so long to review!), we added a requirement to point to sample scan data in parser documentation. This should allow that unit test to pass

```suggestion
~~~

### Sample Scan Data
Sample scan data for testing purposes can be found [here](https://github.com/DefectDojo/django-DefectDojo/tree/master/unittests/scans/noseyparker).
```